assets/grafana/deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       containers:
       - args:
         - -config=/etc/grafana/grafana.ini
-        image: grafana/grafana:6.2.2
+        image: grafana/grafana:6.4.3
```

Comment:
Did we update our grafana as well? 🤔

Reply:
last bump was from june, pinned at 6.2.4, as per https://github.com/openshift/grafana/pull/18

Reply:
(the image is overriden anyways in here, but let's bump it next time)

Reply:
but good catch, I need to add this to my release checklist :+1:

Reply:
I am adding myself as an interested party to maintain downstream grafana as I am already watching what is going on in upstream repository. With that I will take care of bumping our downstream fork periodically.

Reply:
Are you planning to sneakly change to Spartana? @paulfantom =D

Reply:
Maybe :stuck_out_tongue: